src/common/crc32c_aarch64.c

```diff
@@ -178,7 +178,8 @@ uint32_t ceph_crc32c_aarch64(uint32_t crc, unsigned char const *buffer, unsigned
 				"crc32cx        %w[c0],         wzr,    %x[c0]  \n\t"
 				"eor            %w[c],          %w[c],  %w[c0]  \n\t"
 				:[c1]"+r"(crc1), [c0]"+r"(crc0), [c2]"+r"(crc2), [c]"+r"(crc)
-				:[v]"r"(*((const uint64_t *)buffer)));
+				:[v]"r"(*((const uint64_t *)buffer))
+				:"v0" "v1" "v2" "v3");
```

Comment:
just wanted to double check with you, i cannot find the instructions where `v0` or `v1` is specified as the destination register in the inline assembly above. could you point me to them?

Reply:
this file line 144

Reply:
between line 144 and 169，complier may be use v0 and changed the value

Reply:
line 144 is another assembly, which already includes "v0" and "v1" in its clobber list. 

> between line 144 and 169，complier may be use v0 and changed the value

could you be more specific?